.github/workflows/ci.yml

```diff
@@ -0,0 +1,47 @@
+---
+
+name: CI
+
+on:
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+
+jobs:
+  flatpak:
+    name: Flatpak
+    runs-on: ubuntu-latest
+
+    container:
+      image: bilelmoussaoui/flatpak-github-actions:elementary-juno
+      options: --privileged
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Build
+        uses: bilelmoussaoui/flatpak-github-actions/flatpak-builder@master
+        with:
+          bundle: calculator.flatpak
+          manifest-path: io.elementary.calculator.yml
+          run-tests: true
+          repository-name: elementary
+          repository-url: https://flatpak.elementary.io/elementary.flatpakrepo
+          cache-key: flatpak
```

Comment:
this probably won't work due to https://github.com/actions/toolkit/issues/658 & should be as it was.

Reply:
That issue should only happen when the cache does not exist and two runs try to create the cache at the same time. One will get the lock to create the cache, and both will have to run without a cache. This seems pretty normal.

By having a basic cache key, we can rely on the flatpak-builder cache busting instead of our own cache key busting. This is the same thing I did with the platform here: https://github.com/elementary/flatpak-platform/pull/22/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR40

Reply:
> That issue should only happen when the cache does not exist and two runs try to create the cache at the same time. One will get the lock to create the cache, and both will have to run without a cache. This seems pretty normal.

That's not true no. The issue happens even if the cache exists already, we noticed that with OBS hence why I added the possibility to set the cache key to something that changes everytime to avoid the github caching complaining. Note that we already have flatpak & flatpak-builder as secondary cache keys, so you will still hit them.

Reply:
Note that once the upstream issue is resolved, you can completely drop the cache-key from here as the default one should be good enough.